services/docker/docker-version.spec.js

```diff
@@ -48,6 +48,13 @@ describe('DockerVersion', function () {
     }).expect({
       version: '3.10.4',
     })
+    given({
+      tag: '3.9',
+      data: versionDataWithTag,
+      arch: 'arm',
+    }).expect({
+      version: '3.9.5',
+    })
```

Comment:
Could you try updating this test cases to better validate the scenario under test by using a data set that has images that don't all have the same default architecture and the architecture being used for the test? Sure you've seen this already but the data values are defined in `docker-version.spec.js` and if needed we could add another set that has maybe two or three versions, each of which only has one or two images with varying architectures (vs the others that largely have an image for many many archs)

Reply:
Sure, will do this.

Reply:
@calebcartwright Added 3 test cases (2 positive, 1 negative), with the varying architecture data set. Please take a look and let me know if these should be changed / improved.